Preparation/MyStack/MyStackInter.cs

```diff
@@ -33,27 +33,39 @@ public bool IsEmpty()
             return _count < 0;
         }
 
-
         public T Pop()
         {
-            if (_count < 0)
+            if (IsEmpty())
             {
-                Console.WriteLine("Stack is null");
+                throw new InvalidOperationException("Stack is empty!");
             }
             return _underliyngArray[_count--];
         }
 
         public void PrintStack()
         {
-            if (_count < 0)
+            if (IsEmpty())
             {
-                Console.WriteLine("Stack is null");
+                throw new InvalidOperationException("Stack is empty!");
             }
             Console.WriteLine("Your stack is");
             for (int i = _count; i >= 0; i--)
             {
                 Console.WriteLine(_underliyngArray[i]);
             }
         }
```

Comment:
I don;t think this methods is needed here anymore...

Reply:
Which method? IsEmpty or PrintSack?

Reply:
Oh, maybe, I know what about you. It will be correct replace `PrintStack` on `foreach`?

Reply:
Print stack was needed until you implemented for each support. Now it's unneeded.